keyboards/ctrl/ctrl.h

```diff
@@ -0,0 +1,31 @@
+#ifndef _CTRL_H_
+#define _CTRL_H_
+
+#include "samd51j18a.h"
+#include "config.h"
+#include "config_led.h"
+#include "matrix.h"
+
+#define KEYMAP( \
```

Comment:
If you could replace the `KEYMAP` with `LAYOUT`?  And if matches one of the community layouts, that should be used here.